src/components/ebay-date-textbox/component.ts

```diff
@@ -100,7 +101,7 @@ class DateTextbox extends Marko.Component<Input, State> {
     }
 
     openPopover() {
-        this.state.popover = true;
+        this.state.popover = !this.input.disabled && true;
```

Comment:
Is this needed? When the input and button are both disabled, isn't this function unreachable since none of the necessary events are fireable?

Reply:
If this is needed I would change this to an if statemenet rather than assignment. Its much more readable that way.
```
If (!this.input.disabled) {
   this.state.popover = truel;
}
```

Reply:
Good catch, this isn't needed. I removed it.